tests/add_tests.rs

```diff
@@ -75,6 +77,29 @@ async fn add_functionality() {
     ));
 }
 
+/// Test adding a package with a specific channel
+#[tokio::test]
+async fn add_with_channel(){
+    let pixi = PixiControl::new().unwrap();
+
+    pixi.init()
+        .no_fast_prefix_overwrite(true)
```

Comment:
Can't use url's in matchspecs yet, fix in: https://github.com/conda-incubator/rattler/pull/792